### ISSUE TITLE
Task exception was never retrieved

### ISSUE THREAD

**Issue by Daggy1234** [Gorialis/jishaku#65]
### Summary

SO I use sentry.io for error logging and discovered this error happening quite frequently.

```py
Task exception was never retrieved
future: <Task finished name='Task-14762120' coro=<PaginatorInterface.update() done, defined at /usr/local/lib/python3.8/dist-packages/jishaku/paginators.py:272> exception=NotFound('404 Not Found (error code: 10008): Unknown Message')>
```

https://sentry.io/share/issue/48cb426e53e649058eda70f39719eee3/

All the data is at the link above!

**Comment by Gorialis** [comment]
I actually fixed this bug about 4 months ago in e14a6755cf34300f6dc32ad07b771128eb012b3e, but I held off on pushing a new version because I wanted to work on more stuff so I could push a minor version instead of a patch. Since that never ended up happening, this bug has persisted on 1.18.2 even though it was fixed on master long ago.

I've pushed a new patch version and CI/CD is building it now. When it's done, updating jishaku normally to 1.18.3+ should fix this.
